ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -32,8 +32,21 @@ AZURE_SUBSCRIPTION_ID="$(jq -r .data.azure_subscription_id azure_credentials.jso
 rm -f azure_credentials.json
 AZURE_RESOURCE_GROUP=$(oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.azure.resourceGroupName}')
 az login --service-principal -u "${AZURE_CLIENT_ID}" -p "${AZURE_CLIENT_SECRET}" --tenant "${AZURE_TENANT_ID}"
-
-AZURE_VNET_NAME=$(az network vnet list --resource-group "${AZURE_RESOURCE_GROUP}" --query "[].{Name:name}" --output tsv)
+# Recommended on az sites to refresh the subscription
```

Comment:
Minor suggestions for the commit message:

```
In CI we hit problem where just after `az login` the first `az
network vnet list` command fails due to permission. We see
"insufficient permissions" or "pending permissions", suggesting we should
retry later. Manual tests and successful runs indicate we do have the
permissions, but not immediately after login, so add a loop retrying
the first command a few times before declaring failure.
```

Reply:
Using `glop` shows as code in GitHub, the rest is clarifications of what your code does.

Reply:
ack plus I added note about the (later added) `az account set`.